<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IdentityDialog</class>
 <widget class="QDialog" name="IdentityDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>134</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>p2kontakt: New Identity</string>
  </property>
  <widget class="QDateEdit" name="input_date">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>110</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttons_confirm">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>451</width>
     <height>24</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkbox_never">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Never Expire</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_userid">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>User ID (Name)</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkbox_fetch">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>70</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Fetch IPv4 Address</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="tristate">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_ip">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>IP Address</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>input_userid</tabstop>
  <tabstop>input_date</tabstop>
  <tabstop>checkbox_never</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
